### PR TITLE
Remove the age requirement on command roles

### PR DESCRIPTION
**Pull request by sleepyyapril**
## About the PR
Remove the age requirement on command roles.

## Why / Balance
Due to the server rule of needing their character to be at minimum 17 years old (mentally, I assume), this restriction feels incredibly restrictive to people playing borgs with "manufacture date" as the year they were born. In general, this restriction is unneeded.

## Technical details
YML change.

## Media
None

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Removed command age requirement.

### CODE REVIEW

**Review by Colin-Tel — COMMENTED** [review]
Needs Project Manager approval first.

**Comment by BlitzTheSquishy** [comment]
Feels like a non-issue, wouldnt want someone JUST out of highschool running my station, and theres no real reason to up it either

**Comment by Alyx-Wander** [comment]
As a project manager, my own opinion is that if this is to be changed at all, it should be towards the removal of the limit rather than lowering it to an arbitrary amount. It doesn't really add anything mechanically or in terms of roleplay, but does have the ability to exclude players who'm may want to play with non-human life-spans. As an example, synethics who set their age as their date of manufacture, some of the awakened animal characters we have (Soup is an example, although they don't play Command they do play a character with a natural cat's lifespan), and doubtless other examples.

Instead of lowering the requirement to an arbitrary number, we think it'd just be better to outright remove it for the net-zero gain the feature gives us, and the real exclusionary downside it provides as well. Even excluding a handful of players seems like too steep a price for no-benefit.

File: Resources/Prototypes/Roles/Jobs/Cargo/quartermaster.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     - !type:OverallPlaytimeRequirement
       time: 144000 #40 hrs
     - !type:AgeRequirement
-      requiredAge: 20
+      requiredAge: 18
```
Comment (deltanedas) [inline Resources/Prototypes/Roles/Jobs/Cargo/quartermaster.yml]: pro

**Review by deltanedas — CHANGES_REQUESTED** [review]
these all need deltav comments and commenting out instead of removal

**Comment by TadJohnson00** [comment]
Is this still being worked on @flyingkarii ?

**Comment by Samsterious** [comment]
This is one of those debates that is a result of a feature nobody asked for or needed in the first place. Creating an IC age requirement to play a certain role is a strange restriction in a server that includes players 17 and up, and the "highschoolers in command" argument is an easy knee-jerk response but does not actually mean anything in the context of roleplay standards.

The reality is that we have numerous extant characters on this server that are directly harmed by this mechanic: synthetic, genetically-engineered, anomalous, and littlespace characters are all at least partially excluded from Command. These types of characters are most often written by our _best_ roleplayers, not our worst.

Nobody in real life is actually benefiting from this mechanic; its benefits are seen purely within the fiction of the world. It betrays the ethos of Delta-V by excluding a queer identity that is expressed by several of our players, including some of our administration. This PR, or one like it, should be merged _with urgency_.

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
@deltanedas your requested changes are preventing this from being merged and I have no clue why

**Review by deltanedas — APPROVED** [review]

**Pull request merged**